reports/experiments/wrn_28_10_2019-02-06_07-38/wrn_28_10_training_on_cifar10_non_augmented.csv

```diff
@@ -0,0 +1,201 @@
+epoch,acc,loss,val_acc,val_loss

```

Comment:
You might want to add these .csv files to your .gitignore and write them, and all other results the a folder that is not part of your repo.  Results can get take up a lot of space in your repo. This would require that you pass the results folder location as an argument.